src/index.js

```diff
@@ -306,7 +306,10 @@ async function getInput({ entries, cwd, source, module }) {
 		.concat(
 			entries && entries.length
 				? entries
-				: (source && resolve(cwd, source)) ||
+				: (source &&
+						(Array.isArray(source)
+							? source
+							: [source].map(file => resolve(cwd, file)))) ||
```

Comment:
I think you meant to do `(Array.isArray(source) ? source : [source]).map()`?

Reply:
Ah. Yes.